install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -481,6 +481,12 @@ ingressController:
   # Incoming traffic to http listener will return 308 http error code with respective location in header.
   enforceHttps: true
 
+  # -- IngressLBAnnotations are the annotations which are needed to propagate
+  # from Ingress to the Load Balancer
+  ingressLBAnnotationsDefault: ['service.beta.kubernetes.io', 'service.kubernetes.io', 'cloud.google.com']
```

Comment:
we don't need to have one extra fields to set the default value, we can just set ingressLBAnnotations to this array and remove ingressLBAnnotationsDefault.